assemble/contrib/tool.sh

```diff
@@ -26,14 +26,21 @@ contrib=$( cd -P "$( dirname "$SOURCE" )" && pwd )
 basedir=$( cd -P "${contrib}"/.. && pwd )
 # Stop: Resolve Script Directory
 
-source "$basedir"/libexec/load-env.sh
+conf="${basedir}/conf"
+if [ ! -f "${conf}/accumulo.conf" ]; then
+  echo "accumulo.conf must exist in $conf"
+  echo "Run 'accumulo-config' to create it or copy it from $conf/examples"
+  echo "Follow the instructions in INSTALL.md to edit it for your environment."
+  exit 1
+fi
+source "$conf/accumulo.conf"  
 
 if [[ -z "$HADOOP_PREFIX" ]] ; then
-   echo "HADOOP_PREFIX is not set.  Please make sure it's set globally or in conf/accumulo-env.sh"
+   echo "HADOOP_PREFIX is not set.  Please make sure it's set globally or in $conf/accumulo.conf"
```

Comment:
I think this is currently necessary for the property interpolation in the `accumulo-site.xml` file, but I'd really like to phase out having it as a requirement.